packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda"
+import {Logger} from "@aws-lambda-powertools/logger"
+import {injectLambdaContext} from "@aws-lambda-powertools/logger/middleware"
+import {DynamoDBClient, PutItemCommand} from "@aws-sdk/client-dynamodb"
+import {marshall} from "@aws-sdk/util-dynamodb"
+import middy from "@middy/core"
+import inputOutputLogger from "@middy/input-output-logger"
+import errorHandler from "@nhs/fhir-middy-error-handler"
+import {v4 as uuidv4} from "uuid"
+
+const logger = new Logger({serviceName: "updatePrescriptionStatus"})
+const client = new DynamoDBClient({region: "eu-west-2"})
+const tableName = process.env.TABLE_NAME
+
+interface DynamoDBItem {
+  RequestID: string;
+  PrescriptionID: string;
+  PatientNHSNumber: string;
+  PharmacyODSCode: string;
+  TaskID: string;
+  LineItemID: string;
+  TerminalStatusIndicator: string;
+  LastModified: string;
+  Timestamp: string;
+  RequestMessage: any;
+}
+
+const lambdaHandler = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  try {
+    console.log("Received event:", event)
+    const requestBody = JSON.parse(event.body || "")
+    console.log("Parsed request body:", requestBody)
+
+    const entries = requestBody.entry
+    console.log("Entries:", entries)
+
+    if (!entries || entries.length === 0) {
+      console.log("Missing required fields")
+      return {
+        statusCode: 400,
+        body: JSON.stringify({error: "Missing required fields"}),
+        headers: {
+          "Content-Type": "application/fhir+json",
+          "Cache-Control": "no-cache"
+        }
+      }
+    }
+
+    const responseBundle: any = {
+      resourceType: "Bundle",
+      id: uuidv4(),
+      meta: {
+        lastUpdated: new Date().toISOString()
+      },
+      type: "transaction-response",
+      entry: []
+    }
+
+    for (const entry of entries) {
+      console.log("Processing entry:", entry)
```

Comment:
Have a look at https://docs.powertools.aws.dev/lambda/typescript/latest/core/logger/#appending-additional-data-to-a-single-log-item for how to nicely add extra fields to the json log, the way you have them currently structured dont quite behave how i think you think they do "Processing entry:" in this case would be the log message and not the field name.

Reply:
The change has been implemented.
`logger.info("Processing entry", {entry: entry})`